.gitignore

```diff
@@ -141,4 +141,4 @@ dmypy.json
 # Cython debug symbols
 cython_debug/
 
-/test/csv/line of sight analysis.kml
+/test/data/line_of_sight_analysis.kml
```

Comment:
I renamed this folder from `csv` to `data` to avoid conflicts with the `csv` library.